dex/src/main/scala/com/wavesplatform/dex/queue/LocalMatcherQueue.scala

```diff
@@ -112,8 +112,9 @@ object LocalMatcherQueue {
       executor.submit(new Runnable {
         override def run(): Unit = {
           val ts = time.correctedTime()
-          val offset = store.enqueue(command, time.correctedTime())
-          p.success(ValidatedCommandWithMeta(offset, ts, command))
+          store.enqueue(command, time.correctedTime()).foreach { offset =>
```

Comment:
Please use `onComplete` instead of `foreach` on `Futures` and log errors on `Failure`. Also we **don't need to log** `Success`.

Reply:
Done